### PR TITLE
HPCC-16400 Document Keyboard Shortcuts in IDE

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@dehilsterlexis Please review (PDF sent)

### CODE REVIEW

File: docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml [hunk 1]
```diff
@@ -2698,6 +2698,340 @@ ENDMACRO;
...
+                <row>
+                  <entry>Ctrl+O</entry>
+
+                  <entry>Open</entry>
+                </row>
+
+                <row>
+                  <entry>Ctrl+P</entry>
+
+                  <entry>Print</entry>
```
Comment (GordonSmith) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: Not sure we ever supported printing?

File: docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml [hunk 2]
```diff
@@ -2698,6 +2698,340 @@ ENDMACRO;
...
+                <row>
+                  <entry>Ctrl+Shift+Tab</entry>
+
+                  <entry>Previous Tab</entry>
+                </row>
+
+                <row>
+                  <entry>Ctrl+W</entry>
+
+                  <entry>Toggle Horizontal scroll bar</entry>
```
Comment (GordonSmith) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: Should be "Toggle Word Wrap"

**Review by GordonSmith — CHANGES_REQUESTED** [review]
Minor comments

File: docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml (hunk 1 above)
Comment (JamesDeFabia) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: Printing with Ctrl+P works for me :)

File: docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml (hunk 2 above)
Comment (JamesDeFabia) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: Removing this row, Toggle Word Wrap is already listed above

**Comment by JamesDeFabia** [comment]
@GordonSmith Edits made. please look again. Note that ctrl+P does work so I left that in

**Pull request merged**